benchmarks/benchmarks.properties

```diff
@@ -27,6 +27,11 @@
     #sample.sql.db.type=postgresql
     #sample.sql.db.type=sqlserver
 
+    #
+    # Specify whether to use the newly created sites.
+    #
+    sample.sql.group.usage.enable=true
```

Comment:
@slnn Should be `enabled`, see line 330.

Reply:
Done